File: 1-Iniciante/ex3299.c
#include <stdio.h>
#include <string.h>

int main(){
    /*variaveis*/
    int i, tam;
    char numero[19], msg[21];

    /*entrada*/
    scanf("%s", &numero);

    /*processamento*/
    strcpy(msg, "NO es de Mala Suerte");
    tam =  strlen(numero);

    for(i = 0; i < tam; i++){
        if(numero[i] == '1' && i < tam){
            if(numero[i + 1] == '3'){
                strcpy(msg, "es de Mala Suerte");
                break;
            }
        }
    }
    
    /*saida*/
    printf("%s %s\n", numero, msg);

    return 0;
}

File: 1-Iniciante/ex3299.py
#entrada
n = int(input())

#processamento
n = str(n)
msg = 'NO es de Mala Suerte'
for i in range(0, len(n)):
    if n[i] == '1' and i != len(n) - 1:
        if n[i + 1] == '3':
            msg = 'es de Mala Suerte'
            break
#saida
print(n, msg)
